torch_glow/src/PyTorchModelLoader.cpp

```diff
@@ -2729,6 +2744,63 @@ Error PyTorchModelLoader::loadConstant(const torch::jit::Node *ptNode) {
   return Error::success();
 }
 
+Error PyTorchModelLoader::loadEmbeddingBag(const torch::jit::Node *ptNode) {
+  auto inputs = ptNode->inputs();
+  auto outputs = ptNode->outputs();
+  RETURN_IF_ERR(checkInputAndOutputSizes(inputs, 7, outputs, 4));
+
+  glow::NodeValue weight;
+  ASSIGN_VALUE_OR_RETURN_ERR(
+      weight, getGlowNodeValueForValue(inputs[EmbeddingBagInputs::weight]));
+  glow::NodeValue indices;
+  ASSIGN_VALUE_OR_RETURN_ERR(
+      indices, getGlowNodeValueForValue(inputs[EmbeddingBagInputs::indices]));
+  glow::NodeValue offsets;
+  ASSIGN_VALUE_OR_RETURN_ERR(
+      offsets, getGlowNodeValueForValue(inputs[EmbeddingBagInputs::offsets]));
+
+  glow::NodeValue per_sample_weights;
```

Comment:
`perSampleWeights`

This is a weird case because it will basically require two different compiled glow function for if there is or isn't a perSampelWeights